operator-framework-core/src/main/java-templates/io/javaoperatorsdk/operator/api/config/Versions.java

```diff
@@ -0,0 +1,10 @@
+package io.javaoperatorsdk.operator.api.config;
+
+public final class Versions {
+
+  private Versions() {}
+
+  protected static final String JOSDK = "${project.version}";
```

Comment:
Could you pls enlighten me how this supposed to work? Where are those placeholders filled?

Reply:
This file is filled by the Maven Plugin as it is in `java-templates` folder as opposed to `java`.

Reply:
would it be possible to write a unit test for this?

Reply:
great, thx!